### ISSUE TITLE
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.

### ISSUE THREAD

**Issue by reidprichard** [PowerShell/PSReadLine#1586]
Bug was caused by running a short Python 3 script. Cannot be reproduced by re-running the same script under exactly the same conditions.

Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
 n / J F E / r e a d _ s h a r k _ d a t a . p y " Enter
 & Space C : / U s e r s / R e i d / A p p D a t a / L o c a l / P r o g r a m s / P y t h o n / P y t h o n 3 8 / p y t h o n . e x e Space " c : / U s e r s / R e i d / O n e D r i v e Space - Space L i b e r t y Space U n i v e r s i t y / D o c u m e n t s / A c a d e m i c s / R e s e a r c h / S h a r k s k i n / J F E / r e a d _ s h a r k _ d a t a . p y " Enter

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was 20.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)

**Comment by navferty** [comment]
Faced with similar issue. I opened PowerShell window from folder ("open powershell here"), resized it two or three times and tried to view history by pressing up arrow:

```
-----------------------------------------------------------------------Последние клавиши 1:
 UpArrow

Исключение:
System.ArgumentOutOfRangeException: Значение должно быть больше или равно нулю и меньше, чем размер буфера в данной размерности.
Имя параметра: top
Фактическое значение было -1.
   в System.Console.SetCursorPosition(Int32 left, Int32 top)
   в Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   в Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   в Microsoft.PowerShell.PSConsoleReadLine.HistoryRecall(Int32 direction)
   в Microsoft.PowerShell.PSConsoleReadLine.PreviousHistory(Nullable`1 key, Object arg)
   в Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   в Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   в Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
```

Excuse me for localized exception message, but I prefered to copy it as is without modifying.

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.2 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions.
